src/libs/Notification/LocalNotification/BrowserNotifications.js

```diff
@@ -80,7 +82,9 @@ function push({
             }
 
             notification.onclick = (event) => {
-                event.preventDefault();
+                if (getBrowser() !== CONST.BROWSER.FIREFOX) {
+                    event.preventDefault();
+                }
```

Comment:
Curious to know why do we only need it on firefox..

Reply:
@roryabraham I see that you last updated these files. Could you please help me understand the purpose of `event.preventDefault` here? I think it can be removed for all devices.

Reply:
Sorry, I don't know. That line has been here since we first added browser notifications: https://github.com/Expensify/App/pull/289/files#diff-c52e375b9eae87609eec918ff5f478fa0e551f36441417b60bbe089a20567331R83

Reply:
Ok, I tested it. I was assuming that it is used to focus the correct tab which is the active Client when multiple tabs are open. But it is working without it so @K4tsuki just remove this.